src/index.js

```diff
@@ -124,6 +147,47 @@ class Dropzone extends React.Component {
     this.fileInputEl.click();
   }
 
+  walkDirectory(directory, callback) {
```

Comment:
I also think this should be removed from the component code and made a separate module covered with tests.